packages/block-editor/src/components/collab/block-comment-icon-slot.js

```diff
@@ -3,10 +3,10 @@
  */
 import { createSlotFill } from '@wordpress/components';
 
-const { Fill: __unstableCommentIconFill, Slot } = createSlotFill(
-	'__unstableCommentIconFill'
+const { Fill: CommentIconFill, Slot } = createSlotFill(
+	Symbol( 'CommentIconFill' )
 );
 
-__unstableCommentIconFill.Slot = Slot;
+CommentIconFill.Slot = Slot;
```

Comment:
If this is a private API, let's fix it by exporting the return value of `createSlotFill` without any further modifications. We can call it `CommentIconSlotFill` and use its `.Slot` and `.Fill` fields. No needs to have a `Fill` compoennt with a `Fill.Slot` field.

Same thing can be done for `CommentIconToolbarSlotFill`.

Reply:
Updated. I'm sure the names will change again before the feature is stabilized, but it should be good for now.